### DISCUSSION TITLE
Ender 3 v2 homing

### DISCUSSION THREAD

**Discussion by 0xBERNDOG** [xpDIY/3DPlacer#13] [General]
How have you attached the homing extension on the ender 3 v2? I'm considering using tape since mine is a loose fit under the carriage

PS I think a discord server would be nice to have, would be good for low-level chat like this that doesn't require an issue being raised

**Comment by xpDIY** [comment]
Hmm..ok, maybe the tolerance, can you use some tape to attach inside the
openings, then it should be gripping the x-axis metal above the z axis stop
switch.

I am setting up a discord server, should be up soon.

Let me know if that works for you.

xpDIY.io

**Reply by 0xBERNDOG** [reply]
I feeling like there is some variation in the tolerances of the non-critical pieces of these ender printers. Understandable, really
